app/src/main/kotlin/io/homeassistant/companion/android/sensors/GeocodeSensorManager.kt

```diff
@@ -30,6 +30,7 @@ class GeocodeSensorManager : SensorManager {
         const val SETTINGS_INCLUDE_LOCATION = "geocode_include_location_updates"
         private const val DEFAULT_MINIMUM_ACCURACY = 200
         val LOCATION_OUTDATED_THRESHOLD = 5.minutes
+        val GPS_ROLLOVER_WEEKS1024 = (60 * 24 * 7 * 1024).minutes
```

Comment:
```suggestion
        private val GPS_ROLLOVER_WEEKS1024 = (7 * 1024).days
```
Could you make the one above private too 🙏🏻